resource_processor/resources/commands.py

```diff
@@ -48,53 +48,65 @@ async def build_porter_command(config, logger, msg_body, custom_action=False):
             # if still not found, might be a special case
             # (we give a chance to the method above to allow override of the special handeling done below)
             else:
-                parameter_value = get_special_porter_param_value(config, parameter_name, msg_body)
+                parameter_value = get_special_porter_param_value(
+                    config, parameter_name, msg_body
+                )
 
             # only append if we have a value, porter will complain anyway about missing parameters
             if parameter_value is not None:
-                if isinstance(parameter_value, dict) or isinstance(parameter_value, list):
+                if isinstance(parameter_value, dict) or isinstance(
```

Comment:
Adding to previous comment, is this a formatting change do to line length? I'm not sure this updated code is easier to read than the previous one-line `if`.

Reply:
Yeah, I agree that this doesn't really feel easier to read. I assume that it was based on a line-length rule or similar?